akka-docs/src/main/paradox/io-dns.md

```diff
@@ -5,9 +5,13 @@
 `async-dns` does not support:
 
 * [Local hosts file](https://github.com/akka/akka/issues/25846) e.g. `/etc/hosts` on Unix systems
-* [Search domains](https://github.com/akka/akka/issues/25825) e.g. in `/etc/resolve.conf` on Unix systems
 * The [nsswitch.conf](https://linux.die.net/man/5/nsswitch.conf) file (no plan to support)
 
+Additionally, while search domains are supported through configuration, detection of the system configured
+[Search domains](https://github.com/akka/akka/issues/25825) is only supported on systems that provide this 
+configuration through a `/etc/resolv.conf` file, i.e. it isn't supported on Windows or OSX, and none of the 
```

Comment:
There's an `/etc/resolv.conf` on my Mac, though it comes with the header:

```
#
# macOS Notice
#
# This file is not consulted for DNS hostname resolution, address
# resolution, or the DNS query routing mechanism used by most
# processes on this system.
#
# To view the DNS configuration used by this system, use:
#   scutil --dns
#
# SEE ALSO
#   dns-sd(1), scutil(8)
#
# This file is automatically generated.
#
```

(Also s/OSX/macOS/)

Reply:
Yeah, I was aware that that existed. The Netty code that does this implies that Darwin actually has files in `/etc/resolver/*` that can be read, but the documentation it links to 404s.

Reply:
I don't have a `/etc/resolver/` directory at all.  🤷‍♂️